plugins/net_plugin/net_plugin.cpp

```diff
@@ -1642,15 +1644,25 @@ namespace eosio {
 
       sync_reset_lib_num(c);
 
+      auto current_time_ns = std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+      auto network_latency_ns = current_time_ns - msg.time; // net latency in nanoseconds
```

Comment:
Negative would mean time skew between the nodes, should just make it `0` if `< 0` I guess.

Reply:
The check for skew that was removed never worked (see comment I just added to PR for that section of code). I'm open for suggestions on alternatives, but I don't think there is any way to improve that, right?

Reply:
It would have to be based on RTT (which can be measured independent of clock skew) rather than one-way latency.

Reply:
Yes, that would work, but require a new protocol version and RT message.

Reply:
I think it's probably fine to assume low clock skew for now, since we've survived so long without a working check for clock skew.  This PR doesn't make things worse in that regard.